deeplinkdispatch-processor/src/main/java/com/airbnb/deeplinkdispatch/DeepLinkProcessor.java

```diff
@@ -229,7 +251,19 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
     for (AnnotationMirror customAnnotation : annotationMirrors) {
       List<? extends AnnotationValue> suffixes =
           asAnnotationValues(AnnotationMirrors.getAnnotationValue(customAnnotation, "value"));
-      String[] prefixes = prefixesMap.get(customAnnotation.getAnnotationType().asElement());
+
+      Element customElement = customAnnotation.getAnnotationType().asElement();
```

Comment:
it's not clear to me what this has to do with the annotation processing change

Reply:
If someone does not specify the annotation in the build gradle, the annotation will not exist with the map. Without this code change, the processor throws an extremely unhelpful NullPointerException.

This is the point at which the processor is expecting to find the annotation, so it made sense to put the check here to give developers a helpful error telling them they need to add their annotation to the build.gradle

Reply:
Ahhh. I added a test for this scenario now and have found what causes the issue...

Within the code block  `enumerateCustomDeepLinks` it looks for all annotations that meet the criteria (this is able to circumvent the incremental processor only passing back valid annotations)

This only happens when a class is annotated by something else that the annotation processor picks up. i.e.

```
@DeepLink({ "/view_users" })
@WebDeepLink({ "/users", "/user/{id}" })
public class CustomPrefixesActivity {}
```

Even though the WebDeepLink is not registered, the @DeepLink type is. Therefore the class is picked up by the process, it attempts to then find WebDeepLink in the accepted list of annotations, then crashes because it's not there.